.github/workflows/airflow-operator.yml

```diff
@@ -90,15 +90,27 @@ jobs:
       - uses: actions/checkout@v3.3.0
       - run: docker buildx create --name ${DOCKER_BUILDX_BUILDER} --driver docker-container --use
       - run: docker buildx install
-      - uses: actions/setup-go@v4
+
+      - name: Set up Go
```

Comment:
In the test, lint, and release workflows, we use the `magnetikonline/action-galng-cache` plugin, which functionally replaces the functionality of actions/setup-go and actions/cache/restore together - can that work here?

Reply:
Yes - Now that I deleted the extra folder we cache, this can potentially change. But I do think having separate save and cache steps (Which is not possible with `magnetikonline/action-galng-cache`) is better?

Reply:
I don't see the advantage of "separate save and cache steps" with using `actions/cache/restore`  - I believe the `magnetikonline` cache plugin automatically sets up the build and module cache paths as needed before, and persists those when the job is done.